### ISSUE TITLE
Re-export causes transpilation issues from ESM to CommonJS

### ISSUE THREAD

**Issue by jacoblee93** [anthropics/anthropic-sdk-typescript#66]
The export shadowing here: 

https://github.com/anthropics/anthropic-sdk-typescript/blob/main/src/index.ts#L201

Seems to result in issues with compilation from ESM to CommonJS. Anything that could be changed in this SDK to help?

Context:
https://github.com/hwchase17/langchainjs/issues/1958

Am pushing a workaround for now but it's not nice (and TBH I don't fully understand the interaction, but I think it's because namespaces don't get picked up in CommonJS?):

https://github.com/hwchase17/langchainjs/pull/1969/files#diff-52e43107f1daefd085360ec9360a7ba2235d8d24c052ba8ff945867e90abcba9R18

This is our cjs tsconfig file if it helps, not terribly complex:
```
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "module": "commonjs",
    "declaration": false
  },
  "exclude": [
    "node_modules",
    "dist",
    "docs",
    "**/tests"
  ]
}
```

**Comment by dqbd** [comment]
Digging a little deeper into the weeds, it seems like the culprit is the addition of `exports = module.exports = Anthropic`, which will remove the occurrence of `Anthropic` export in the `module.exports` for CJS but not for ESM.

https://github.com/anthropics/anthropic-sdk-typescript/pull/29/commits/ae4591908d2c52a9be8982764aa1735df6da69cd#diff-7e95e175bdcd8908ec8a97efb902828d36225f3a70a73dcca0139950487036ccR8

```
> const { Anthropic } = require("@anthropic-ai/sdk");
> console.log(Anthropic)
< undefined
```

```
> import { Anthropic } from "@anthropic-ai/sdk";
> console.log(Anthropic)
< [class Anthropic extends APIClient]...
```

**Comment by jedwards1211** [comment]
@dqbd yup, we need to add `static Anthropic = this` to the `Anthropic` class so that both `require('@anthropic-ai/sdk')` and `require('anthropic-ai/sdk').Anthropic` work correctly

**Comment by rattrayalex** [comment]
This has been fixed in v0.5.6 which is available on GitHub now and will be published to npm soon.
